features/async-function.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Async functions
+category: js
+bugzilla: 1185106
+firefox_status: in-development
```

Comment:
Indeed it seems so: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_Next_support_in_Mozilla